android/app/src/main/res/values/attrs.xml

```diff
@@ -2,7 +2,9 @@
 <resources>
   <declare-styleable name="SpeedLimitView">
     <attr name="BackgroundColor" format="color" />
+    <attr name="WidgetBackgroundColor" format="color" />
```

Comment:
What's the difference between `BackgroundColor` and `WidgetBackgroundColor`?
It should be possible to understand it from the name.
Btw, why do these values have names starting from upper case and values below have names starting from lower case?
They all should start from lower case letters like any other android's XML value.